bin/reportdaily.py

```diff
@@ -120,6 +264,19 @@ def parsecli(cliargs=None) -> argparse.Namespace:
 
     # subparser
     subparsers = parser.add_subparsers(help='available sub commands')
+    # init cmd
+    parser_init = subparsers.add_parser(
+        'init', help="Create an initial Configuration file")
+    parser_init.set_defaults(func=cmd_init)
+    parser_init.add_argument(
+        '--name', "-n", help='This is a option for the user to enter his name direct via console')
+    parser_init.add_argument(
+        '--year', "-y", help='This is a option for the user to enters his teaching year direct via console')
+    parser_init.add_argument(
+        '--team', "-t", help='This is a option for the user to enter his team year direct via console')
+    parser_init.add_argument(
+        '--change', '-c', action='store_true', help='This is a option for the user to change an existing configuration')
```

Comment:
It's a bit hard to follow if the help starts with "This is a option [for the user]...". Of course it is! :wink: Also "via console" is stating the obvious.

So in the first case, I'd remove all the obvious things and reduce it to the bare minimum:

    User name

Shorten the other help options in a similar way.